### DISCUSSION TITLE
De-Duplicated imports

### DISCUSSION THREAD

**Discussion by emhl** [IgnisDa/ryot#1041] [Ideas]
if a play with the same timestamp already exists it would be nice if it wouldn't be imported again.

**Comment by IgnisDa** [comment]
Sorry de-duplicated imports would be very difficult to implement and will require a lot of changes. I don't plan to add that feature.

What's your use case? Imports are meant to be a one-time operation, so it doesn't make sense to implementation deduplication.

**Reply by emhl** [reply]
i didn't have ryot running for a while so i wanted to import the plays from trakt etc. that weren't recorded by ryot
And importing from trakt and jellyfin directly will inevitably lead to duplictes as well. but they will both have plays that only exist on one of the sites as well

**Reply by emhl** [reply]
allowing actions on multiple plays (with an easy selection of all plays) would allow fixing such situations manually as well.
